### PR TITLE
chore (video): update tests in the samples for video intelligence

### PR DESCRIPTION
**Pull request by viacheslav-rostovtsev**
Make tests in the samples for video intelligence use Minitest instead of rspec

### CODE REVIEW

**Comment by viacheslav-rostovtsev** [comment]
last fix is 'by feel' xref https://github.com/seattlerb/minitest-focus/pull/16 by @dazuma seems like minitest/focus needs to be explicitly required or else

File: video/acceptance/video_samples_test.rb [hunk 1]
```diff
@@ -13,121 +13,103 @@
...
-    }.to output(
-      /Label description: animal/
-    ).to_stdout
+    end
   end
 
   it "can analyze labels from a local file" do
     begin
-      local_tempfile = Tempfile.new "temp_video"
+      local_tempfile = Tempfile.new "temp_video#{SecureRandom.hex}"
```
Comment (dazuma) [inline video/acceptance/video_samples_test.rb:40]: You do not need the `SecureRandom`. The `Tempfile` class automatically generates a unique name in the temporary directory.

**Comment by dazuma** [comment]
gimmeproj is fixed. One comment remains above.

File: video/acceptance/video_samples_test.rb (hunk 1 above)
Comment (viacheslav-rostovtsev) [reply video/acceptance/video_samples_test.rb]: fixed

**Pull request merged**